examples/adaptiveprecision-blockjacobi/CMakeLists.txt

```diff
@@ -1,5 +1,12 @@
-set(target_name "adaptiveprecision-blockjacobi")
-add_executable(${target_name} ${target_name}.cpp)
-target_link_libraries(${target_name} Ginkgo::ginkgo)
-target_include_directories(${target_name} PRIVATE ${PROJECT_SOURCE_DIR})
+cmake_minimum_required(VERSION 3.9)
```

Comment:
Ginkgo itself requires 3.13 now. I guess there is no need, but you could upgrade this as well.